docs/examples.md

```diff
@@ -31,128 +42,94 @@ new CheckmarxSecurityJobBuilder(
 ```
 import jenkins.automation.builders.BddSecurityJobBuilder
    
-   def projectName ='foo'
-   def bddSecurityRepo ="repo-to-scan'
-   new BddSecurityJobBuilder(
-           name: "${projectName}bdd_security_job",
-           description: "Sample bdd security job",
-           baseUrl: "http://google.com",
-           bddSecurityRepo: "${bddSecurityRepo}",
-           chromedriverPath: "\\/Users\\/sotoo\\/homebrew\\/bin\\/Chromedriver"
-   ).build(this);
-   
-
-```
-
-## Base Job Builder
-
-
-```
-import jenkins.automation.builders.BaseJobBuilder
-   
-   def baseJob = new BaseJobBuilder(
-                   name: this.name,
-                   description: this.description,
-                   emails: this.emails 
-   ).build(this)
-
+def projectName ='sample-project'
+def bddSecurityRepo ="repo-to-scan'
+new BddSecurityJobBuilder(
+       name: "${projectName}-bdd-security-job",
+       description: "Sample bdd security job",
+       baseUrl: "http://google.com",
+       bddSecurityRepo: "${bddSecurityRepo}",
+       chromedriverPath: "\\/Users\\/sotoo\\/homebrew\\/bin\\/Chromedriver"
+).build(this)
 ```
 
 ## Flow Job Builder
 
-
 ```
 import jenkins.automation.builders.FlowJobBuilder
 
-    def flowJob= new FlowJobBuilder(
-            name: 'GeneratedFlowJob',
-            description: 'this our first stab at it',
-            jobs:['job1', 'job2']
-    ).build(this);
-    
-    flowJob.with{
-      logRotator{
-          numToKeep(365)
-      }
-    
-    }
-    def customFlowJob= new FlowJobBuilder(
-            name: 'GeneratedCustomFlowJob',
-            description: 'this a custom flow job',
-            jobFlow: """
-                build('job1')
-                build('job2')
-                parallel(
-                 { build('job3') },
-                 { build('job4') },
-                )
-            """
-    ).build(this);
-
-    customFlowJob.with{
-      logRotator{
-          numToKeep(365)
-      }
-
-    }
+FlowJobBuilder(
+        name: 'sample-flow-job',
+        description: 'A sample build flow job that builds a list of jobs in order',
+        jobs:['job1', 'job2']
+).build(this).with {
+  logRotator{
+      numToKeep(365)
```

Comment:
I think it's useful to at least show how... I don't think that's exactly encouraging it